### PR TITLE
De-duplicated and re-arranged basic rules

### PR DESCRIPTION
**Pull request by bleehu**
added Primary and Secondary Stats to glossary
removed some feats rules from feats document, added feats section to basic rules
re-wrote secondary stats section
fixed b0rk'd headings not diplaying properly

More work is yet to be done; in particular, the character creation section is still duplicated. But I figured this already had enough scope creep to get bogged, so I'll continue in another PR.
	modified:   01_01_Basic_Rules.txt
	modified:   03_Glossary_of_Terms.txt
	modified:   CharacterCreation/03_Feats.txt

### CODE REVIEW

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: lang (sc)- I'd swap this to "time and place of your adventure is up to..." b/c it reads less awk

Comment (rosebrallier) [reply 01_01_Basic_Rules.txt]: s-v agreement: "adventures"

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "friends and foes"

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: lang- "how well they perform different tasks"

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:304]: New Issue(?): Oops. I haven't been dropping the remainder before multiplying when moving for two actions.
Honestly I like my way better.

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: *"could have"

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: *"Primary Stats"

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:393]: "primary stat points"

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "react quickly" (reason- parallel structure)

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: Breaks the parallel structure altogether :(

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: *five; "feats, detailed below" (reason: maintains 2nd person)

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:478]: I'd nix the first sentence of Passive, Active and Weapon feats and make a note in leveling up about which feats can be chosen when. Or specify "When you gain a non-Class feat upon leveling up..."

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "while a character with 60 points would..."

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: @Turtlelord26 Does it make sense that Unarmed is the pre-req for Melee? I'd think that Unarmed Expertise should be its own thing, and the pre-req for Melee should be proficiency with a melee weapon type.

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "like Active, Passive or Weapon Expertise feats"

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: In the interest of making space for future developments how about "Stat requirement" as opposed to "Luck or Charisma"?

**Review by rosebrallier — APPROVED** [review]
I threw in a bunch of wording suggestions for funsies.
Thank you for all the necessary but less thrilling work here @bleehu !

**Comment by bleehu** [comment]
I cannot tell you how much I want to just merge this and move on. But If you're going to take the time to do a full review of the document, the least I can do is make the corrections. Will execute and then merge.

**Comment by bleehu** [comment]
@rosebrallier, made the changes. Want me to wait for another review, or just shipit?

**Comment by rosebrallier** [comment]
Shipit!!!!!! (So many the thanks!)

Rose Brallier

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:449]: This is a fantastic solution!!!

File: 01_01_Basic_Rules.txt
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: I love the Oxford comma!

**Review by rosebrallier — APPROVED** [review]

File: 01_01_Basic_Rules.txt
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt:478]: Second the removal of the first sentences.

File: 01_01_Basic_Rules.txt
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt:304]: Fortunately, the rules do not mandate dropping any remainders when moving 2 actions. The base unit is Move Speed per 2 Actions, and the division only happens when moving for exactly 1 Action.
You have been correct

File: 01_01_Basic_Rules.txt
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt:508]: Nice to see these in this document 🐢

**Review by Turtlelord26 — APPROVED** [review]
Very, very shiny!

Aside from two remaining nb inlines from Rose, very shipit the thing!

**Pull request merged**
